client/toDataByLocation.js

```diff
@@ -1,4 +1,4 @@
-export default function toDataByLocation(data, datasetKey) {
+module.exports = function toDataByLocation(data, datasetKey) {
```

Comment:
do you know why you needed to change this? Does our jest setup include babel? https://jestjs.io/docs/en/getting-started#using-babel

Reply:
just wanna follow up here, do you know why this was necessary?